elastic/apm-server
# fix: add missing timestamp to agent_config metric

## Motivation/summary

Adds the missing timestamp to the `agent_config_applied` metricset.

## Checklist



~- [ ] Documentation has been updated~

## How to test these changes

1. Ensure `app-server` is running in managed mode via the fleet integration.
1. Make config changes to the apm config through fleet.
1. Ensure the `agent_config` metric sets have a timestamp that is not `0001-01-01T00:00:00.000Z`.

## Related issues

Closes #6379
